pkgs/top-level/all-packages.nix

```diff
@@ -22480,6 +22480,10 @@ in
 
   vscodium = callPackage ../applications/editors/vscode/vscodium.nix { };
 
+  vscode-oss = callPackage ../applications/editors/vscode-oss {
+    electron = electron_7;
```

Comment:
Doesn't this work with `electron_8`? If not, please include link to the upstream GitHub issue.

Reply:
Original target was set to electron `7.11.1`, while it probably works with 8, I did want to provide most compatible build.

Reply:
I am going through the whole nixpkgs tree and it seems it's more common just require the electron_7 directly in your vscode-oss package. I will send the PR to remove this usage in the outstanding 2-3 or packages which do it this way and add unnecessary extra lines to all-packages.nix.

Reply:
I don't know if that's good. What if I want to use electron_8 instead of 7 and override derivation. It becomes a bit wierd overriding electron_7 with electron_8. The same happens if I am overriding src with never vscode that requires electron_8.

Reply:
As vscode potentially runs with never versions of electron, I think we should override in `all-packages.nix` or change default electron to never (most compatible) version.

Reply:
I am not strongly opposing to change attribute name, just felt like this is kind of more right. Originally I was using `electron_7`, but changed before pushing.

Reply:
This should be now solved.